ros2interface/package.xml

```diff
@@ -26,8 +26,11 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_xmllint</test_depend>
+  <test_depend>launch</test_depend>
+  <test_depend>launch_ros</test_depend>
```

Comment:
Technically, I don't think this package directly depends on launch_ros.

Reply:
6f5a060